### PR TITLE
Align with our own guidance for codeowners team name

### PR DESCRIPTION
**Pull request by andyblundell**

### CODE REVIEW

**Comment by andyblundell** [comment]
As things stand we have 2 codeowners files, but let's make sure the new team works before we do anything about that

**Comment by stefaniuk** [comment]
@andyblundell Sorry, I'm not sure why I haven't spotted the one in the root directory. Would it be a good idea to move it to the `.github` directory so anything specific to GitHub functionality can be placed there in the dedicated folder? Just a thought.

**Comment by andyblundell** [comment]
@stefaniuk good call - I *think* a copy in .github takes precedent over a copy in the root folder, so yep .github is the better place for it. Looking at your copy again, I think we've ended up with a global code owner (from the original file) plus some specific code owner patterns (from your file). Let's revert to a single file (now in .github), but I'm not sure if those new patterns are actually doing anything though, given they all point to the same team as the global entry?

**Comment by andyblundell** [comment]
I've removed the CODEOWNERS file from the root folder, having copied the default owner pattern to the .github copy of the file

But unless I'm missing something the new patterns are redundant, since they all point to the same team as the default pattern...

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,8 @@
-/.github @NHSDigital/quality-framework-admins
-/.vscode @NHSDigital/quality-framework-admins
-/.gitignore @NHSDigital/quality-framework-admins
-/project.code-workspace @NHSDigital/quality-framework-admins
+# Default owners for everything in the repo - unless a later match takes precedence
+@NHSDigital/software-engineering-quality-framework-code-owners
```
Comment (stefaniuk) [inline .github/CODEOWNERS:2]: We may need a '*' character at the beginning of this line if we would like to enforce review of any change in the repo be the 'code-owners'.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+
+# Owners for specific areas in the repo
```
Comment (stefaniuk) [inline .github/CODEOWNERS:4]: Yes, that's right. If the above is in place that block is redundant.

**Review by stefaniuk — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (andyblundell) [reply .github/CODEOWNERS]: Silly me - thanks :)

File: .github/CODEOWNERS (hunk 1 above)
Comment (andyblundell) [reply .github/CODEOWNERS]: My previous "silly me" was supposed to be here :)

File: .github/CODEOWNERS (hunk 2 above)
Comment (andyblundell) [reply .github/CODEOWNERS]: I'v removed those extra blocks - we can add more later if we do want different owners for different sections of the project

**Review by iw-ezequiel — APPROVED** [review]

**Pull request merged**